portfolio/src/main/webapp/images/photography/fileNames.txt

```diff
@@ -0,0 +1,13 @@
+1-dalis.jpg	    1-valley1.jpg      3-plane.jpg	     4-vivocity2.jpg
+1-dog1.jpg	    2-bridge.jpg       3-waterfall.jpg	     4-vivocity3.jpg
+1-dog2.jpg	    2-farm.jpg	       3-wave.jpg	     5-dexposed1.jpg
+1-fog.jpg	    2-forest.jpg       4-blk71.jpg	     5-dexposed10.jpg
+1-halo.jpg	    2-horse.jpg        4-jewel1.jpg	     5-dexposed2.jpg
+1-house.jpg	    2-silhouette2.jpg  4-jewel2.jpg	     5-dexposed3.jpg
+1-houses1.jpg	    2-sky1.jpg	       4-jewel3.jpg	     5-dexposed4.jpg
+1-houses2.jpg	    2-sky2.jpg	       4-jewel4.jpg	     5-dexposed5.jpg
+1-lembangfault.jpg  2-sunrise1.jpg     4-jewel5.jpg	     5-dexposed6.jpg
+1-mountains.jpg     2-sunrise2.jpg     4-lightleak.jpg	     5-dexposed7.jpg
+1-people.jpg	    3-boats.jpg        4-mrt.jpg	     5-dexposed8.jpg
+1-silhouette.jpg    3-cliff.jpg        4-universitytown.jpg  5-dexposed9.jpg
+1-sunrise.jpg	    3-mountain.jpg     4-vivocity1.jpg	     fileNames.txt
```

Comment:
should fileNames.txt be in here? Also, where is this file being used? If not used, you can remove it :)

Reply:
I was just using it to populate an array! I'll be deleting all unused files :)